### DISCUSSION TITLE
Adding highlightjs?

### DISCUSSION THREAD

**Discussion by chrisxfire** [hugo-sid/hugo-blog-awesome#58] [Q&A]
Hi,

I'm trying to add syntax highlighting support for `razor` and `cshtml` files to my blog.  My current approach is to use [highlightjs](https://github.com/highlightjs/highlightjs-cshtml-razor)'s cshtml-razor JavaScript file.  They have made it available via CDN:

```html
<script type="text/javascript" src="https://unpkg.com/highlightjs-cshtml-razor/dist/cshtml-razor.min.js"></script>
```

Is it possible to add this script tag somewhere to hugo-blog-awesome in order to enable syntax highlighting support for razor and cshtml files?

**Comment by hugo-sid** [comment]
Hi Christian (@chrisxfire ).

> Is it possible to add this script tag somewhere to hugo-blog-awesome in order to enable syntax highlighting support for razor and cshtml files?

You can include the script in `layouts/partials/scriptsBodyEnd.html` .

**Reply by chrisxfire** [reply]
Hi,

Yes, that would work.  It seems that HBA (or perhaps Hugo in general?) uses chroma for syntax highlighting.  I suspect I'm having issues because either Hugo or this theme is using chroma and ignoring highlight.js... any chance you could point me in the right direction to solve for this?
